test/include/autoinstall/AutoinstallAskInclude_test.rb

```diff
@@ -0,0 +1,292 @@
+#!/usr/bin/env rspec
+
+root_path = File.expand_path('../..', __FILE__)
+ENV["Y2DIR"] = File.join(root_path, 'src')
+
+require "yast"
+require_relative "../../../src/include/autoinstall/ask.rb"
+require_relative "../../../src/modules/Profile"
+
+describe "Yast::AutoinstallAskInclude" do
+  module DummyYast
+    class AutoinstallAskClient < Yast::Client
+      def main
+        Yast.include self, "autoinstall/ask.rb"
+      end
+
+      def initialize
+        main
+      end
+    end
+  end
+
+  let(:client) { DummyYast::AutoinstallAskClient.new }
```

Comment:
I use a `let` because subject does not refers exactly to the module (the module is included in the dummy client I'm using to peform the tests). To make it clear, we also could use the subject(:client) syntax [described by David Chelimsky (at the end)](http://blog.davidchelimsky.net/blog/2012/05/13/spec-smell-explicit-use-of-subject/). Thoughts?